multinet/uploaders/swagger/csv.yaml

```diff
@@ -34,6 +34,21 @@ parameters:
     schema:
       type: boolean
       default: false
+  -
+    name: metadata
+    in: query
+    description: Metadata to set for the table
+    schema:
+      type: object
+      example:
+        columns:
+          -
+            key: city
+            type: label
+          -
+            key: size
+            type: number
```

Comment:
I know this broken item style is allowed, but elsewhere in our YAML files (even in other ones in this PR) we don't use it. For consistency I'd prefer to remove these linebreaks.

Reply:
It's used elsewhere in this file, that's why I wrote it this way:

https://github.com/multinet-app/multinet-server/blob/6ef8fdca1aee350fa18395c398a5f454e5c1ff4e/multinet/uploaders/swagger/csv.yaml#L9-L36

Reply:
Ok. Then don't worry about it.